src/theory/bags/infer_info.h

```diff
@@ -72,6 +74,14 @@ class InferInfo : public TheoryInference
    * engine with no new external premises (d_noExplain).
    */
   bool isFact() const;
+  /**
+   * @return the lemma for this InferInfo.
+   */
+  Node getLemma() const;
+  /**
+   * @return whether this InferInfo represents a cached lemma.
+   */
+  bool isCachedLemma() const;
```

Comment:
Can probably just delete this too right?

Reply:
Done.